apps/build.gradle

```diff
@@ -60,18 +61,16 @@ subprojects { subproject ->
         }
     }
 
-    def packageJson = file("package.json").text
-
-    task lint(type: YarnTask, dependsOn: format) {
-        if (packageJson.contains("\"name\": \"tailwind-components\"")) {
-            args = ['run', 'lint']
+    task buildLibs(type: YarnTask, dependsOn: format) {
+        if (packageJson.contains("\"name\": \"tailwind-components\"") || packageJson.contains("\"name\": \"molgenis-components\"")) {
+            args = ['run', 'build']
             inputs.files fileTree(projectDir).matching {
                 with excludeDef
             }
         }
     }
 
-    task buildJavascript(type: YarnTask, dependsOn: lint) {
+    task buildJavascript(type: YarnTask, dependsOn: buildLibs) {
```

Comment:
Might not fix issue if the dependsOn only applies "within its own street of parallelization" where each individual `app` is a separate parallel process.

In that case it would result in `tailwind-components:buildJavascript` dependsOn `tailwind-components:buildLibs`, but not `catalogue:buildJavascript` dependsOn `tailwind-components:buildLibs`

Might need to be more specific? See f.e.
https://github.com/molgenis/molgenis-emx2/blob/a8f257fb9242488b913510f8d1acc69a0e2542d7/apps/build.gradle#L76-L79